# Array of detected obstacles
std_msgs/Header header
Obstacle[] obstacles
int32 total_obstacles  # Total number of obstacles detected